R/dataframe.R

```diff
@@ -76,7 +76,7 @@ lst_ <- function(xs) {
     deparse2 <- function(x) paste(deparse(x$expr, 500L), collapse = "")
     defaults <- vapply(xs[missing_names], deparse2, character(1),
       USE.NAMES = FALSE)
-
+    defaults <- ifelse(defaults == "NULL", "", defaults)
```

Comment:
I leave it in your hands. I did it to match the naming behaviour of `list()`. Was trying to avoid naming unnamed NULLs `"NULL"`.